### ISSUE TITLE
f-droid version 1.2 crashes

### ISSUE THREAD

**Issue by jakobdettner** [zouroboros/filmchecker#3]
When I add a new dm film the app crashes.
Version 1.1 from f-droid works fine.
Tested with oneplus5 android8.0.0

**Comment by zouroboros** [comment]
At first thanks for taking your time and reporting this. I am really happy that some else uses this app. 

I pushed some changes that should hopefully resolve your problem. It would be cool if you could comment on weather that fixed your problem or not. (It make take one or two days until f-droid shows you the new version).

**Comment by jakobdettner** [comment]
Thank you for your time. I will report back if it works.

**Comment by jakobdettner** [comment]
Hi, I've installed the latest version 1.2.2 from fdroid. First it crashed when I started the app but after cleaning the data and cache it was working.
But now when I try to add a new DM film I cannot insert the 'Filialnummer' anymore. Instead I have to give a 'HT-NR.' which I don't know what it is:

![screenshot_20180314-211650](https://user-images.githubusercontent.com/29184219/37428790-a8b9d002-27cd-11e8-8b9d-5552efe089a6.jpg)

When I enter the Filialnummer in the HT-NR. field and hit save it says: "Store number does not exist. [VSN]"
